src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT_FIGURES = 5;
+    private static final int MAX_RANDOM_PARAMETR = 100;
+    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    private static final int DEFAULT_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(COUNT_FIGURES);
+        switch (index) {
+            case 0:
+                return getRandomSquare();
+            case 1:
+                return getRandomRectangle();
+            case 2:
+                return getRandomRightTriangle();
+            case 3:
+                return getRandomCircle();
+            default:
+                return getRandomIsoscelesTrapezoid();
+        }
+    }
+
+    private Square getRandomSquare() {
+        return new Square(colorSupplier.getRandomColor(), random.nextInt(MAX_RANDOM_PARAMETR));
```

Comment:
create separate method for `random.nextInt(MAX_RANDOM_PARAMETR)` and use it for each figure